lib/libc/minimal/source/stdlib/abort.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
```

Comment:
shouldn't this be declared in stdlib.h (./lib/libc/minimal/include/stdlib.h)

Reply:
> shouldn't this be declared in stdlib.h

Ack, fixed.